addons/mail/static/src/models/message_seen_indicator/message_seen_indicator.js

```diff
@@ -18,6 +18,7 @@ function factory(dependencies) {
          * @param {mail.thread|integer} thread
          * @returns {string|undefined}
          */
+        // FIXME replace by using messageId & channelId as identifying fields (task-2335647)
```

Comment:
strange position to place the comment, either keep it as single line and move inside the method, or keep it above and move into dostring